docs/trace/building-your-own-exporter/MyExporter.cs

```diff
@@ -27,6 +27,10 @@ internal class MyExporter : ActivityExporter
     public override Task<ExportResult> ExportAsync(
         IEnumerable<Activity> batch, CancellationToken cancellationToken)
     {
+        // Exporter code which can generate further
+        // telemetry should do so inside SuppressInstrumentation
+        // scope. This suppresses telemetry from
+        // exporter's own code to avoid live-loop situation.

```

Comment:
Should we explain more here?

Reply:
I think we chatted about this [here](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1006#discussion_r465483708).

I would vote for not having comments to keep the code short (e.g. having 50 lines of code for a dummy exporter seems too much). No strong opinion though 😄

Reply:
I strongly feel we should leave some comments about the SupressInstrumentation part, as this is not a spec defined name.